src/pages/on-chain-governance/[proposalId]/index.page.tsx

```diff
@@ -37,6 +44,8 @@ export default function ProposalDetailPage({
     VoteStages.VoteProposal
   );
 
+  const [isCurrentCycleClicked, setIsCurrentCycleClicked] = useState(true);
```

Comment:
- Current cycle only applies to the context of CFP. Rename it to something more self-explanatory, e.g.: `cfpVotingResultCycleTab`
- One very important note, do not leave gray area for states, now you're assuming handling currentCycle = true state for VOC proposal too, let's not do that, handle them explicitly
- Avoid from using boolean flag for tab-related states, even though it is only just two tabs, using flag may trigger questions like what is the other tab?